src/sysctl/sysctl.c

```diff
@@ -279,10 +280,10 @@ static int parse_line(const char *fname, unsigned line, const char *buffer, bool
         return 0;
 }
 
-static int parse_file(OrderedHashmap **sysctl_options, const char *path, bool ignore_enoent) {
+static int parse_file(OrderedHashmap **sysctl_options, const char *path, bool ignore_enoent, const char **search_paths) {
         return conf_file_read(
                         /* root = */ NULL,
-                        (const char**) CONF_PATHS_STRV("sysctl.d"),
```

Comment:
ugh? why? this adds one memory allocation for no good reason?

Reply:
the memory allocation don't matter too much. we should avoid them when easy. but i do see the benefit of having this all hooked into the same source of truth here, and i do see the value of exposing sysctl.d/ the same way as binfmt.d/ and tmpfiles.d/ and so on, because this is inherently also a concept of accepting config provided by other packages.

so i am inclined to merge this.